modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Grid/Grid.es.js

```diff
@@ -75,11 +75,13 @@ const Grid = ({
 	onBlur,
 	onChange,
 	onFocus,
+	required,
 	rows = [{label: 'row', value: 'jehf'}],
 	value,
 	...otherProps
 }) => (
 	<div className="table-responsive" {...otherProps}>
+		<fieldset aria-label="grid" aria-required={required}>
```

Comment:
@rijema, do we need the `<fieldset>`? Won't it change the UI like below:
<img width="701" alt="Screen Shot 2020-07-29 at 14 09 02" src="https://user-images.githubusercontent.com/6729287/88830986-13c97400-d1a5-11ea-9c39-68536eded000.png">
A UI change like that, at a first moment, isn't intended.

In addition, will the value of the property `aria-label` be used by a screen reader? If so, then we should consider the language to set the proper attribute's value.

Reply:
Hey @natocesarrego ! No, it doesn't change the UI. Here below I'm adding a picture of the current state of the form with the new commits. Couple months ago we also added this tag,[ you can see it here:](https://github.com/natocesarrego/liferay-portal/pull/422#issuecomment-607425716)

I didn't make any commit about the comment above.

![85BE5CAB-A443-4D89-B06F-2FDDC756B153](https://user-images.githubusercontent.com/41641596/89057312-cc252280-d333-11ea-9687-6a5db8c61106.jpeg)

Yes, the value will be used by the ScreenReader. To explain what was my thought: I looked for other aria-label on others fields, and I found aria-label="text" and aria-label="numeric".
So, in that way, I tried to follow the pattern.

I added a new commit making the change to use the properly label value. Let me know if you want me to update this pull request with a new commit or if I have to squash in some of these.